adapter/integration-test/src/adapter-datasource.test.js

```diff
@@ -160,19 +147,19 @@ describe('Adapter Configuration', () => {
     expect(eventsAfter.body[0].eventId).toBe(eventId + 1)
     expect(eventsAfter.body[0].datasourceId).toBe(datasourceId)
     expect(eventsAfter.body[0].eventType).toEqual('DATASOURCE_DELETE')
-  })
+  }, TIMEOUT)
 
   test('GET datasources/events/latest', async () => {
```

Comment:
I did not change the event based tests since I am going to remove them in the next pull request anyways

Reply:
Do you remove all tests?

Reply:
I want to replace the http event publishing by amqp. Therefore all tests related to the event http endpoints will be replaced by rabbitmq stuff